.github/workflows/ci.yml

```
@@ -5,34 +5,40 @@ on: [ push, pull_request ]
 jobs:
    build:
       runs-on: ubuntu-latest
+      outputs:
+         nvmrc: ${{ steps.makeNodeVersionOutput.outputs.nvmrc }}
       steps:
-         -
-            uses: actions/checkout@v3
+         -  uses: actions/checkout@v4
```

Unless it's a single key, we typically put the keys on separate lines:

```suggestion
         -
            uses: actions/checkout@v3
```